src/client.ts

```diff
@@ -61,6 +61,9 @@ class ApiError extends Error {
   }
 }
 
+// We should retry request only in case of timeout or disconnect
+const RETRY_CODES = ['ECONNABORTED', 'ECONNREFUSED'];
```

Comment:
make this into a `Set` so that later on, you can do `RETRY_CODES.has(code)` (quicker lookup)